docs/data/data-grid/pagination/PageSizeInitialState.tsx

```diff
@@ -17,7 +17,7 @@ export default function PageSizeInitialState() {
           ...data.initialState,
           pagination: {
             ...data.initialState?.pagination,
-            pageSize: 25,
+            paginationModel: { pageSize: 25, page: 0 },
```

Comment:
It should be the default value
```suggestion
            paginationModel: { pageSize: 25 },
```